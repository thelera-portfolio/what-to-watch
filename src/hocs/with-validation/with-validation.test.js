import * as React from "react";
import renderer from "react-test-renderer";
import withValidation from "./with-validation";

const MockComponent = () => <article />;

const MockComponentWrapped = withValidation(MockComponent);

it(`withValidation is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      validationMessage={`message`}
      onValidForm={() => {}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
