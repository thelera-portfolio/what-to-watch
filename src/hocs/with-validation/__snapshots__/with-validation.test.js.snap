// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withValidation is rendered correctly 1`] = `<article />`;
